doc/DataPlaneCodeGeneration/EvolutionArchBoard.md

```diff
@@ -4,183 +4,567 @@
+|MetricsAdvisorClient.AddFeedback| POST | one required parameter | Single Object | [API](https://apiview.dev/Assemblies/Review/a9812a1691e54ab4bf52540c4e54d433?diffRevisionId=9260bdbf10e346b9ba8dac7399ea885b&doc=False&diffOnly=False&revisionId=66cd7343ae6c47988e3f8b3caf52617b#Azure.AI.MetricsAdvisor.Models.DataFeed) L12, 13| [Code](https://github.com/ShivangiReja/azure-sdk-for-net/blob/812720991ec401f51f8bdb958781d48d6edd24e4/sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/MetricsAdvisorClient.cs#L220)|
+|MetricsAdvisorAdministrationClient.GetDataFeed| GET | one required paramter | Single Object | [API](https://apiview.dev/Assemblies/Review/a9812a1691e54ab4bf52540c4e54d433?diffRevisionId=9260bdbf10e346b9ba8dac7399ea885b&doc=False&diffOnly=False&revisionId=66cd7343ae6c47988e3f8b3caf52617b#Azure.AI.MetricsAdvisor.Models.DataFeed) L231, 232 | [Code](https://github.com/ShivangiReja/azure-sdk-for-net/blob/812720991ec401f51f8bdb958781d48d6edd24e4/sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/MetricsAdvisorAdministrationClient.cs#L78)|
+|DeviceUpdateClient.GetOperation| GET | ETag parameter | Single Object | [API](https://apiview.dev/Assemblies/Review/4d7d53c6581245759f465e12c8d9e5c5) L86, 88| [Code](https://github.com/chunyu3/azure-sdk-for-net/blob/10765f008e541992e1f20fc72d46a5165bc59a3a/sdk/deviceupdate/Azure.IoT.DeviceUpdate/src/DeviceUpdateClient.cs#L69)|
 
-```csharp
+#### Scenario: no required parameters
+
+If your convenience method has the same parameter list as that of the protocol method, the initial version of convenience method could have ambiguity with the protocol method. To resolve that issue we will append suffix (such as `Value` or `Values`) to the initial method name as the convenience method name.
+
+- Protocol Method
+
+```c#
+public virtual Response GetConstitution(RequestContext context = null)
+{
+    using var scope = ClientDiagnostics.CreateScope("ConfidentialLedgerClient.GetConstitution");
+    scope.Start();
+    try
+    {
+        using HttpMessage message = CreateGetConstitutionRequest(context);
+        return _pipeline.ProcessMessage(message, context);
+    }
+    catch (Exception e)
+    {
+        scope.Failed(e);
+        throw;
+    }
+}
+public virtual async Task<Response> GetConstitutionAsync(RequestContext context = null)
+{
+    using var scope = ClientDiagnostics.CreateScope("ConfidentialLedgerClient.GetConstitution");
+    scope.Start();
+    try
+    {
+        using HttpMessage message = CreateGetConstitutionRequest(context);
+        return await _pipeline.ProcessMessageAsync(message, context).ConfigureAwait(false);
+    }
+    catch (Exception e)
+    {
+        scope.Failed(e);
+        throw;
+    }
+}
+
+```
+
+- Convenience Method
+
+```c#
+public virtual Response<LedgerConstitution> GetConstitutionValue(CancellationToken cancellationToken = default)
+{
+    Response response = GetConstitution(new RequestContext { CancellationToken = cancellationToken });
```

Comment:
For example, this implementation is incorrect, because we would want to add a diagnostic scope that uses the public method name and suppress the inner scope.